### ISSUE TITLE
How to install new icons

### ISSUE THREAD

**Issue by agarcialeon** [Chris2011/Visual-Studio-Code-Iconset-for-NetBeans#1]
Is there any instruction to change the defaul file icons with these?

**Comment by Chris2011** [comment]
This repo never gets enough attention to work on it. This was just a map for different filetypes and folders to add new icons. What you can do for like a new icon for new file action, you can see here: https://blogs.oracle.com/geertjan/icon-sets-for-netbeans-ide. The Problem is, we need to find out all those instances from NetBeans and map them to a specific icon. And not only for specific actions, also for filetypes etc. There is atm no real solution for that to do it like VS Code does. You can ask at the mailing list.

**Comment by Chris2011** [comment]
This approach will be very cool: https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons but I don't know how to do this. Each file, that NetBeans knows is done by creating a new fileType with an icon, extension and mimetype. So you need to read all fileObjects or smth like that and set the icon new to them.

Just guessing.

**Comment by agarcialeon** [comment]
Yep, I was searching for something similar to this. Will have to investigate more.

**Comment by Chris2011** [comment]
That sounds cool :) I invited you into our slack chan.
